frontend/src/components/news-page/NewsCard.tsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import { RedditVideo } from "../../clients/redditApiClient";
+import { Card } from "../card/Card";
+import "./NewsCard.scss";
+
+interface NewsCardProps {
+  video: RedditVideo;
+}
+
+export const NewsCard: React.FC<NewsCardProps> = ({ video }) => {
+  return (
+    <Card title={video.title}>
+      <div className="video-container">
+        <video
+          key={video.media.reddit_video.fallback_url}
```

Comment:
I think you can drop the `key` from this component, since `key` is only necessary inside a mapped list!
```suggestion
```
(I know I copied and pasted this code myself, but I missed it 😉)